### PR TITLE
The Boss Is In

### PR DESCRIPTION
**Pull request by ITCups**
## About The Pull Request
<details>
<summary>
	New IH perks
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
add: New IH perks

/:cl:

### CODE REVIEW

**Comment by LethalGhost** [comment]
Are you sure perks should be part of stat system?

File: code/datums/perks/perk.dm [hunk 1]
```diff
@@ -80,3 +80,11 @@
 
 /datum/perk/proc/is_active()
 	return active
+
+/mob/proc/has_perk(perk_type)
+	if(!stats)
+		return FALSE
+	for(var/datum/perk/P in stats.perks)
+		if(P.type == perk_type)
```
Comment (PlinHost) [inline code/datums/perks/perk.dm]: You should use `stats.getPerk(perk_type)` or at least `locate(perk_type) in stats.perks`.

**Comment by ITCups** [comment]
> Are you sure perks should be part of stat system?

Its already in, this question should have been addressed to @PlinHost

**Comment by LethalGhost** [comment]
> 
> 
> > Are you sure perks should be part of stat system?
> 
> Its already in, this question should have been addressed to @PlinHost

Okay.

**Comment by SpadesSlick** [comment]
Delayed until documentation on combat system rework will arrive (soon).

**Pull request closed without merging**